### ISSUE TITLE
removeCombined does not removed ordered files

### ISSUE THREAD

**Issue by kohenkatz** [dbashford/mimosa-combine#7]
Given the following configuration:

```
  combine:
    folders: [
      {
        folder:'Content'
        output:'Content/styles.css'
        order: ['bootstrap.css', 'bootstrap-responsive.css']
        exclude: [/\.svg$/]
      }
      {
        folder:'Scripts'
        output:'Scripts/vendor.js'
        order: ['jquery-1.9.1.js', 'knockout-2.2.1.js']
      }
    ]
```

and the following folder structure:

```
./
    assets/
        Content/
            app.css
            bootstrap.css
            bootstrap-responsive.css
            durandal.css
            font-awesome.css
        Scripts/
            bootstrap.js
            jquery-1.9.1.js
            knockout-2.2.1.js
            sammy-0.7.4.js
    public/ (This is the build directory)
```

After the files have been combined, `bootstrap.css`, `bootstrap-responsive.css`, `jquery-1.9.1.js`, and `knockout-2.2.1.js` are not removed from the public directory, despite the fact that they are included in the combined styles/scripts and they are not set to be excluded.

This appears to be because of [Line #69](https://github.com/dbashford/mimosa-combine/blob/master/src/index.coffee#L69), which removes the ordered files from the set so they don't get included twice.

There are two immediately apparent solutions, but I'm not sure which is better:
- Work with a copy of `folderFiles` and keep the original that is returned from the function untouched.
- Re-add the ordered files to `folderFiles` before returning from the function.

**Comment by dbashford** [comment]
Thanks for checking out mimosa and for the bug report!  Just about done fixing this up, will get a new version of `mimosa-combine` out in a bit.

**Comment by dbashford** [comment]
Released as `mimosa-combine@0.8.0`.  Either refer to `0.8.0` specifically in your `mimosa-config` or nuke the `mimosa-combine` folder in your `node_modules` directory.

Let me know if you have any trouble!
